worker/src/features/evaluation/evalService.ts

```diff
@@ -195,6 +201,217 @@ export const createEvalJobs = async ({
   }
 };
 
+const datasetEval = async ({
+  event,
+  configs,
+}: {
+  event: z.infer<typeof DatasetRunItemUpsertEventSchema>;
+  configs: JobConfigurationsRaw;
+}) => {
+  logger.info(
+    `Creating eval jobs for dataset run item ${event.datasetItemId} on project ${event.projectId}`,
+  );
+
+  for (const config of configs) {
+    if (config.status === "INACTIVE") {
+      logger.debug(`Skipping inactive config ${config.id}`);
+      continue;
+    }
+
+    // safeguard only, this should never happen
+    if (config.target_object !== "dataset") {
+      logger.debug(
+        `Skipping eval config ${config.id} as target object is not dataset run item`,
+      );
+      continue;
+    }
+
+    logger.info("Creating eval job for config", config.id);
```

Comment:
If we bring this live, we need to exclude these logs from the DD log index? https://github.com/langfuse/infrastructure/blob/388b4a65de116742eb08a979dfb9ea58301b36d5/terraform/org/infrastructure/datadog_log_archive.tf#L3

Reply:
This has been live since we released model based evals... And can you give me context on why we'd need to exclude these logs from the DD index?